scripts/lib/desktop-rtl-lib.ps1

```
@@ -2273,14 +2547,40 @@ function Test-RtlOwnedCommand {
+        if ($targetsAgent) {
+            $ev = $null
+            try {
+                $ev = New-Object System.Threading.EventWaitHandle($false, [System.Threading.EventResetMode]::ManualReset, $script:AgentQuitEventName)
+                [void]$ev.Set()
+                $deadline = [DateTime]::Now.AddSeconds($GraceSec)
+                while ([DateTime]::Now -lt $deadline) {
+                    $alive = @($victims | Where-Object { Get-Process -Id $_.ProcessId -ErrorAction SilentlyContinue })
+                    if ($alive.Count -eq 0) { break }
+                    Start-Sleep -Milliseconds 200
+                }
+            } finally { if ($ev) { try { $ev.Dispose() } catch {} } }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)